SECURITY.md

```diff
@@ -1,22 +1,29 @@
 # Security Policy
 
-## Supported Versions
+This Command-Line HTTP Server is intended to be as configurable as possible,
+thus security concerns may vary based on the configuration provided on runtime.
 
-Use this section to tell people about which versions of your project are
-currently being supported with security updates.
+### Data Security
 
-| Version | Supported          |
-| ------- | ------------------ |
-| 0.5.x   | :white_check_mark: |
-| 0.4.x   | :x:                |
-| 0.3.x   | :x:                |
-| 0.2.x   | :x:                |
-| 0.1.x   | :x:                |
+If you are running this server to send/receive sensitive data, please make sure
+you configure TLS support and provide a certificate.
+
+Follow on [TLS/HTTPS configuration][1] for more details.
+
+### Host Security
+
+This software is primary written in Rust. Rust is well known for its security
+principles by design, access to invalid memory addresses or invalid references
+are mostly reduced. Even though these warranties many not apply in some
+use cases (those where `unsafe` code is introduced), this software tends to
+avoid the usage of these code blocks. If you find any space for replacement
+of `unsafe` code and you are not sure on how could you apply it, please don't
+hesitate on contacting software editors by opening an issue or a pull request.
 
 ## Reporting a Vulnerability
 
-Use this section to tell people how to report a vulnerability.
+If you have found any issue using this solution, please don't hesitate to
+open a Pull Request or send an email to estebanborai@gmail.com if you consider
+the issue of urgent priority.
```

Comment:
I think generally the norm would be to email privately, so that the bug doesn't become public knowledge until it is already fixed and on crates.io/lib.rs/whatever. Also, you might want to use a dedicated email to lessen spam to your ordinary email.

Reply:
> I think generally the norm would be to email privately, so that the bug doesn't become public knowledge until it is already fixed and on crates.io/lib.rs/whatever.

Totally agree!

> Also, you might want to use a dedicated email to lessen spam to your ordinary email.

I agree with you, but as of today I don't mind to use personal email. I'm also working on setting up a email address for software support using my personal domain, but until is available and in order to have progress with this, I would start by using my personal email.